INTERCHANGE_FORMAT.md

```diff
@@ -0,0 +1,133 @@
+<img src="https://raw.githubusercontent.com/css-modules/logos/master/css-modules-logo.png" width="150" height="150" />
+
+# CSS Interchange Format (CIF)
+
+This document describes the specification of the low-level file format that enabled CSS Modules. This is designed for loader-implementers, not for end-users. For the high-level specification, see the full [CSS Modules](https://github.com/css-modules/css-modules/blob/master/README.md) spec.
+
+## Rationale
+
+As JavaScript workflows have trended towards building collections of components, CSS workflows have followed suit. However, any progress on the CSS front has been purely *conventional*, not supported by the language. The most visible example of this is the [BEM methodology](http://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/), but the argument is common to many approaches:
+
+- Styles should be scoped to a single component
+- All CSS selectors are global
+- ∴ Develop a convention for ensuring globally-unique selectors
```

Comment:
I had no idea, thank you.

On Thu, Jun 4, 2015 at 5:26 PM, Glen Maddern <notifications@github.com>
wrote:

> In INTERCHANGE_FORMAT.md
> <https://github.com/css-modules/css-modules/pull/5#discussion_r31705330>:
>
> > @@ -0,0 +1,133 @@
> > +<img src="https://raw.githubusercontent.com/css-modules/logos/master/css-modules-logo.png" width="150" height="150" />
> > +
> > +# CSS Interchange Format (CIF)
> > +
> > +This document describes the specification of the low-level file format that enabled CSS Modules. This is designed for loader-implementers, not for end-users. For the high-level specification, see the full [CSS Modules](https://github.com/css-modules/css-modules/blob/master/README.md) spec.
> > +
> > +## Rationale
> > +
> > +As JavaScript workflows have trended towards building collections of components, CSS workflows have followed suit. However, any progress on the CSS front has been purely *conventional*, not supported by the language. The most visible example of this is the [BEM methodology](http://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/), but the argument is common to many approaches:
> > +
> > +- Styles should be scoped to a single component
> > +- All CSS selectors are global
> > +- ∴ Develop a convention for ensuring globally-unique selectors
>
> Nope, it's just shorthand for "therefore"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/css-modules/css-modules/pull/5/files#r31705330>.
>